### DISCUSSION TITLE
Implementation of firebase in gdsc website

### DISCUSSION THREAD

**Discussion by Yashsharma1911** [GDSCITM/GDSCITM-official-website#110] [Ideas]
We are near to introducing firebase implementation on our website @ArunPatel02 can you explain how we can go ahead also would you mind creating an issue with the task in it that need to be implemented

// FYI @Soumya-Kushwaha @kamalchakma1 @martinyis @mukadas026

**Comment by ArunPatel02** [comment]
Hey everyone we are planning to integrate Firestore / real time database of Firebase for fetching and creating the data for 
1-> Events that will be organized by GDSC.
2-> Team members 
as we are integrating Firebase, there is no need for much static data. so, we need to add data in events and teams data automatically. we do not want to add data manually through the Firebase website.

so, can anyone have any suggestions on how we can proceed? or we should introduce some forms to post data?

we really appreciate all of your suggestions.

// FYI @kamalchakma1 @Soumya-Kushwaha  @kamalchakma1  @martinyis  @mukadas026  @Yashsharma1911  @sudhanshutech @dhamkirti21 @masif2002  @GDSCITM/contributors

**Reply by masif2002** [reply]
Yea, I think we create a separate web application for admins to add EventData, EventManagers and so on.. Since it is only meant for admins, I don't think it would make sense to expose the app to the public

**Reply by kamalchakma1** [reply]
@masif2002 is right , I also think we should make one admin page separately.

**Reply by Yashsharma1911** [reply]
thanks for your suggestion guys
